Plugins/SwiftProtobufPlugin/plugin.swift

```diff
@@ -18,15 +18,50 @@ struct SwiftProtobufPlugin: BuildToolPlugin {
             /// The visibility of the generated files.
             enum Visibility: String, Codable {
                 /// The generated files should have `internal` access level.
-                case `internal`
+                case `internal` = "Public"
```

Comment:
Sadly no. I wanted to have test that executes this plugin; however, to do so it needs protoc. Since we haven't solved the "where does protoc" come from problem yet we really can't put an example target into this package.